django/http/response.py

```diff
@@ -37,7 +37,6 @@ class HttpResponseBase(six.Iterator):
     """
 
     status_code = 200
```

Comment:
Perhaps we can use the same approach for status_code and status attributes, too.

Setting two different attributes to do the same job would be bad:

```py
resp = HttpResponse(status=503)

resp = HttpResponse()
resp.status_code = 503
```

status_code is already used in many places, perhaps we can also add status_code to the constructor of HttpResponseBase.